packages/ui-causal-graph-editor/src/shared/graph-layout/common.tsx

```diff
@@ -46,6 +48,22 @@ export abstract class GraphLayoutBuilder<T> {
     abstract build(): T;
 }
 
+export interface TiersConfig {
```

Comment:
Make sure to export all of the tier types from the package so they can be imported later in `dara`.

Reply:
Good point, fixed